SWLOR.Game.Server/Service/LanguageService.cs

```diff
@@ -127,7 +127,7 @@ public static string TranslateSnippetForListener(NWObject speaker, NWObject list
             lastSkillUp = (lastSkillUp << 32) | (uint)lastSkillUpLow;
             long differenceInSeconds = (now - lastSkillUp) / 10000000;
 
-            if (differenceInSeconds / 60 >= 5)
+            if (differenceInSeconds / 60 >= 2)
```

Comment:
This makes it fire once every 2 minutes instead of 5 right? If so, I'm down with this change.

Reply:
Yeah, this was decided over just increasing the amounts because this will help the Comprehend Speech Force power as well because you only trigger xp on that when the language triggers for XP.